### PR TITLE
adding new blog post on jlap

### PR DESCRIPTION
**Pull request by travishathaway**
### Description

This is a blog by @dholth on the new JLAP format. It was originally posted on [https://conda.discourse.group](https://conda.discourse.group).

### CODE REVIEW

**Comment by tnabtaf** [comment]
@travishathaway Think about inserting a

`<!-- truncate -->`

at some point.  That will prevent the whole post from showing up on the blog index page.  (And it will insert a read more link.)

**Comment by travishathaway** [comment]
@tnabtaf,

Thanks for the recommendation! Didn't know you could do that. I will update my other blog post here too (the "Welcome to conda dot org" one).

**Comment by travishathaway** [comment]
I'm also adding this to the new conda build post too.

**Review by jezdez — APPROVED** [review]
I've read this before and think it's a great example of a personal blog post, nicely done!

Just had a question of embedding the asciicinema vidoo directly?

File: blog/2023-05-05-how-we-reduced-the-conda-index-fetch-bandwidth.mdx [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+
+When the cache is empty, conda will try to download `repodata.json.zst`. This file is much faster to download and decompress compared to `Content-Encoding: gzip` and is slightly smaller.
+
+When the cache is primed, conda will look for `repodata.jlap`. It will download the entire file, apply any relevant patches (comparing to a content hash of `repodata.json`) and remember the length of the patch file.
+
+On subsequent fetches, conda will use a HTTP Range request to download new bytes, if any, added to `repodata.jlap`, and apply new patches.
+
+## Conclusion
+
+[This screen capture of a conda-forge/noarch search](https://asciinema.org/a/574800) shows that were able to download a single update to the channel in 1464 bytes for what would have otherwise have been a 10358799-byte download of the complete index. The patch size is proportional to the amount of change that has happened since the last time you ran conda.
```
[the remark this answers is not in the document]
Comment (jaimergp) [reply blog/2023-05-05-how-we-reduced-the-conda-index-fetch-bandwidth.mdx:105]: We need to define an `Asciinema` component, like this: https://consolelogdupa.com/embedding-asciinema-asciicast-in-your-docusaurus-2-docs

[the remark this answers is not in the document]
Comment (travishathaway) [reply blog/2023-05-05-how-we-reduced-the-conda-index-fetch-bandwidth.mdx:105]: Sounds like a good feature request for future work. Let's just leave it the way it is for now.

**Pull request merged**